### PR TITLE
DNM: Test KCP with loopback client fix

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vince@prigna.com>

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by vincepri** [comment]
@ncdc Tests pass, the PR is based on https://github.com/kcp-dev/kubernetes/pull/121

File: pkg/virtual/apiexport/builder/forwarding.go [hunk 1]
```diff
@@ -67,6 +67,7 @@ func provideDelegatingRestStorage(ctx context.Context, clusterClient kcpdynamic.
 			typer,
 			namespaceScoped,
 			kind,
+			nil,
```
Comment (ncdc) [inline pkg/virtual/apiexport/builder/forwarding.go:70]: @sttts is `nil` ok here (and in the other files below)?

**Comment by ncdc** [comment]
@vincepri I merged your k8s PR but I'd like to hear from @sttts re the new `kcpValidateName` field that you're setting to `nil` now.

**Comment by ncdc** [comment]
@vincepri let's wait to hear from Stefan

**Pull request closed without merging**